web/pom.xml

```diff
@@ -46,10 +47,6 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
-    <dependency>
-      <groupId>com.azure.spring</groupId>
-      <artifactId>spring-cloud-azure-starter-jdbc-postgresql</artifactId> <!-- For passwordless connection -->
-    </dependency>
```

Comment:
If we remove this, does it mean it doesn't have the PostgreSQL driver?

Reply:
yes, it should be, and I am wondering how does it run successfully?

Reply:
Actually it failed in enterprise tier, but it succeeded in consumption or standard tier. I guess the data didn't get written to the PSQL db in that two tiers.

Reply:
seems it is not using postgres:
https://github.com/Azure-Samples/ASA-Samples-Web-Application/blob/1653c9f99812e97ed608aafb615a94f5a1c8fcb7/web/src/main/resources/application.yml#L1-L3

Reply:
Oh, I was wrong, consumption plan failed too.

Reply:
But why, if using azd, will it add service connection to PSQL?

Reply:
This mem db should be used when in local, but with PSQL on Azure, it should use the env var for SPRING_DATASOURCE_URL.